docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -266,8 +266,51 @@ Type | Key | Value | Default value | Description
 &#xfeff;                                                     |                             | relu                        |                         | Relu function
 &#xfeff;                                                     |                             | softmax                     |                         | Softmax function
 &#xfeff;                                                     | cell_state_zoneout_rate     | (float)                     | 0                       | Zoneout rate for cell state
-&#xfeff;                                                     | hidden_state_zoneout_rate   | (float)                     | 0                       | zoneout rate for hidden state
+&#xfeff;                                                     | hidden_state_zoneout_rate   | (float)                     | 0                       | Zoneout rate for hidden state
 &#xfeff;                                                     | integrate_bias              | (boolean)                   | false                   | Integrate bias_ih, bias_hh to bias_h
+`ML_TRAIN_LAYER_TYPE_ATTENTION` (since 7.0)                  |                             |                             |                         | Attention layer
+&#xfeff;                                                     | n/a                         |                             |                         |
+`ML_TRAIN_LAYER_TYPE_MOL_ATTENTION` (since 7.0)              |                             |                             |                         | Mol attention layer
+&#xfeff;                                                     | unit                        | (unsigned integer)          |                         | Number of output neurons
+&#xfeff;                                                     | MoL_K                       | (unsigned integer)          |                         | Size of the three projections
+`ML_TRAIN_LAYER_TYPE_MULTI_HEAD_ATTENTION` (since 7.0)       |                             |                             |                         | Multi head attention layer
+&#xfeff;                                                     | num_heads                   | (unsigned integer)          | 1                       | Number of head
+&#xfeff;                                                     | projected_key_dim           | (unsigned integer)          |                         | projected key dim per head
```

Comment:
This is minor thing but can we start with capital letter? Cause other descriptions are also started with capital letter.
projected .. => Projected ...

Reply:
thank you.